resources/metadata/karmaExecuteTests.yaml

```
@@ -31,9 +31,9 @@ spec:
           - STAGES
           - STEPS
         mandatory: true
-      - name: modulePath
+      - name: modules
         type: string
```

This should be a list as in Groovy:

https://github.com/SAP/jenkins-library/blob/6805654cdb620a6036c0a55f0acc268ca50d3807/resources/default_pipeline_environment.yml#L316-L317

```suggestion
        type: "[]string"
```